translations/ar.json

```diff
@@ -1,3 +1,3 @@
 {
-  "Terra.devSiteRoutingMenu.filter": "منقي"
+  "Terra.devSite.themed.help": "المساعدة"
```

Comment:
We no longer support arabic translations. I think we could remove this

Reply:
it's just for demo purposes, i borrowed it from somewhere else.